server/db/controllers/users.js

```diff
@@ -1,8 +1,6 @@
-import User from '../models/users';
-
 export default {
-  saveNewUser: params =>
-    new User({
+  saveNewUser: (params, Model) =>
```

Comment:
This is just my preference, but I'd provide the `{}` for each method.

export default {
    saveNewUser: (params, Modal) => {
        const newUser = new User()...
        // Can also do some logging/metrics here
        return newUser;
    },

}